configuration/device-tree.md

```diff
@@ -463,20 +539,19 @@ dtparam=foo=bar,level=42
 
 This will cause the loader to look for `overlays/acme-board.dtbo` in the firmware partition, which Raspbian mounts on `/boot`. It will then search for parameters `foo` and `level`, and assign the indicated values to them.
 
-The loader will also search for an attached HAT with a programmed EEPROM, and load the supporting overlay from there; this happens without any user intervention.
+The loader will also search for an attached HAT with a programmed EEPROM, and load the supporting overlay from there - either directly or by name from the "overlays" directory; this happens without any user intervention.
 
 There are several ways to tell that the kernel is using Device Tree:
 
 1. The "Machine model:" kernel message during bootup has a board-specific value such as "Raspberry Pi 2 Model B", rather than "BCM2709".
-2. Some time later, there may also be another kernel message saying "No ATAGs?" - this is expected.
-3. `/proc/device-tree` exists, and contains subdirectories and files that exactly mirror the nodes and properties of the DT.
+2. `/proc/device-tree` exists, and contains subdirectories and files that exactly mirror the nodes and properties of the DT.
 
 With a Device Tree, the kernel will automatically search for and load modules that support the indicated enabled devices. As a result, by creating an appropriate DT overlay for a device you save users of the device from having to edit `/etc/modules`; all of the configuration goes in `config.txt`, and in the case of a HAT, even that step is unnecessary. Note, however, that layered modules such as `i2c-dev` still need to be loaded explicitly.
 
-The flipside is that because platform devices don't get created unless requested by the DTB, it should no longer be necessary to blacklist modules that used to be loaded as a result of platform devices defined in the board support code. In fact, current Raspbian images ship without a blacklist file.
+The flipside is that because platform devices don't get created unless requested by the DTB, it should no longer be necessary to blacklist modules that used to be loaded as a result of platform devices defined in the board support code. In fact, current Raspbian images ship with no blacklist files except for a few WLAN devices where multiple drivers are available.
```

Comment:
Would it be appropriate to put the "except ..." part at the end in brackets? :man_shrugging:

Reply:
Spose so.